events/message.js

```diff
@@ -3,6 +3,24 @@ const { prefix, colors } = require('../config.json')
 
 const reactMessage = require('../utils/reactMessage')
 
+cmdQueue = []
+handlingCommand = false
+
+var waitToRunCommand = function() {
+  setTimeout(runCommand, 300) 
```

Comment:
Why is there a timeout in the queue?

Reply:
There was an error where the bot would leave the voice channel and then reenter it immediately, which would cause the command to stop running and the bot got stuck in the voice channel, so I buffered it with that.